packages/react/src/hooks/useAuth.ts

```diff
@@ -27,10 +27,10 @@ export const useAuth = (): UseAuthResult => {
     user: undefined,
   });
 
-  const handleAuth = ({ payload }) => {
+  const handleAuth: HubCallback = ({ payload }) => {
     switch (payload.event) {
       case 'signIn':
-        return setResult({ user: payload.data, isLoading: false });
+        return setResult({ user: payload.data as AuthUser, isLoading: false });
```

Comment:
Do we still need the type assertion after typing the function? That's annoying

Reply:
Yeah unfortunately, `data` is typed as `any` in `HubCapsule`